tmpl/button-row.php

```diff
@@ -14,7 +14,14 @@
 	<div class="cookie-consent-message">
 		<?php echo wp_kses_post( $banner_message ); ?>
 		<?php if ( $policy_page ) : ?>
-			<?php load_template( __DIR__ . '/cookie-consent-policy.php' ); ?>
+			<?php
+				/**
+				 * Allow the cookie consent policy template path to be overridden so it can be customized individually. This template displays the link to the cookie policy page, but only if a cookie policy page has been set.
+				 *
+				 * @var string The path to the cookie consent policy template.
+				 */
+				apply_filters( 'altis.consent.cookie_consent_policy_template_path', load_template( __DIR__ . '/cookie-consent-policy.php' ) );
```

Comment:
Hmmm, will this work? I think that if you use that filter to add a different template, it will load anyway the default one as `load_template( __DIR__ . '/cookie-consent-policy.php' )` will be always executed and will echo that template. If you want to filter the template path, I think what you wanted could be:

```suggestion
				load_template( apply_filters( 'altis.consent.cookie_consent_policy_template_path', __DIR__ . '/cookie-consent-policy.php' ) );
```
But you also may want to check if the template exists as if the filter returns a path that doesn't exist, it could end up in  a fatal.

Reply:
Generally it's more readable to set a variable from the `apply_filters()` call rather than try to use it directly inline. It's easier to add a docblock to the filter as well then.

Reply:
@roborourke Agree, I would also prefer a variable but the thing here is that the filter was useless as the template was anyway loaded.

Reply:
@igmoweb yep. brain not working. 👍 makes perfect sense.